src/components/Advertisements/Advertisements.tsx

```
@@ -127,57 +131,61 @@ export default function Advertisements(): JSX.Element {
+                  loader={
+                    <>
+                      {/* Skeleton loader while fetching more advertisements */}
+                      {[...Array(32)].map((_, index) => (
+                        <div key={index} className={styles.itemCard}>
+                          <div className={styles.loadingWrapper}>
+                            <div className={styles.innerContainer}>
+                              <div
+                                className={`${styles.orgImgContainer} shimmer`}
+                              ></div>
+                              <div className={styles.content}>
+                                <h5 className="shimmer" title="Name"></h5>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)